cmd/kubeadm/.import-restrictions

```diff
@@ -73,7 +73,6 @@
 				"k8s.io/kubernetes/pkg/scheduler/api",
 				"k8s.io/kubernetes/pkg/scheduler/nodeinfo",
 				"k8s.io/kubernetes/pkg/scheduler/internal/cache",
-				"k8s.io/kubernetes/pkg/scheduler/util",
```

Comment:
I think that one of our tools will complain if the new package name is not added to this list. See https://github.com/kubernetes/kubernetes/pull/68968#discussion_r222766882

Reply:
I think we're on the same page. This PR is meant to be a package rename, so it's not a new import. However, it looks like @mysunshine92 actually removed a dependency in this PR, so adding the new name to this file will not be needed.

Reply:
> I think we're on the same page. This PR is meant to be a package rename, so it's not a new import. However, it looks like @mysunshine92 actually removed a dependency in this PR, so adding the new name to this file will not be needed.

ah, the [latest push](https://github.com/kubernetes/kubernetes/compare/7479f62d435d51dce2a6bbf5ad89b66407ef90e2..1b3f577cb80a077e907e7f5ef30774e9d3e0bd7a#diff-36c8b8a331cd03a763c5ab0a33f97acc) removed the internal package my comment referred to

Reply:
@liggitt  @misterikkit  so now this import is ok? i should not add  new  name to this file? thank you

Reply:
removal of  the `k8s.io/kubernetes/pkg/scheduler/util` import is fine. adding a new internal import is not ok, so if renaming the package required that, this PR also needs to make adjustments to avoid kubeadm and kubectl depending on a new internal package. those adjustments can be made in a second commit